packages/visual-group/src/group-helper/cell-creator.js

```diff
@@ -445,6 +447,22 @@ export const generateMatrices = (context, matrices, cells, labelManager) => {
         rowPriority
     };
 };
+const getAxisFields = (projections, fieldHolder = []) =>
+                            projections.reduce((acc, item) =>
+                                [...acc, ...item.reduce((ac, field) =>
+                                   (field instanceof SimpleVariable ? [...ac, field.oneVar()] : ac), [])], fieldHolder);
+
+const sortDmTemporalFields = (resolver, datamodel) => {
+    let axisFields = [];
+    const projections = resolver.projections();
+    axisFields = getAxisFields(projections.colProjections, axisFields);
+    axisFields = getAxisFields(projections.rowProjections, axisFields);
```

Comment:
In both the lines, axisFields = getAxisFields...
Two possible cases:

1) Either the first value of axis fields gets overridden. In this case, the first line is redundant
**OR**
2) axisFields gets mutated in the getAxisFields Function. In which case, the equation doesnt make sense. Only an expression is required

Please address the redundant line/expressions